manifests/l/LAIPIC/Perfoo/v1.0.4/LAIPIC.Perfoo.installer.yaml

```
@@ -0,0 +1,32 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: LAIPIC.Perfoo
+PackageVersion: v1.0.4
```

Automation needs to remove the `v`
```suggestion
PackageVersion: 1.0.4
```